### DISCUSSION TITLE
Support for the gdb protocol

### DISCUSSION THREAD

**Discussion by tstih** [maziac/DeZog#118] [General]
Hi,

[the documentation](https://github.com/maziac/DeZog/blob/main/design/EmulatorInterface.md#mame) says that MAME uses gdb protocol which was not implemented. But MAME is listed in the supported platforms. 

Is my assumption that the documentation is a bit outdated and that DEZOG now speaks GDB protocol correct? 

If not, I know GDB protocol well and would be interested in writing the stand-alone adapter GDB<->DZRP.

Regards,
Tom

**Comment by maziac** [comment]
Hello Tom,
Thanks for pointing me to that documentation.
Your assumption is correct: the documentation is a little outdated.
I have to update it.
Meanwhile I implemented the bridge DZRP to gdb.
I.e. MAME is supported through gdb.

Thanks for volunteering anyway.
